studybuddies/src/app/app.module.ts

```diff
@@ -18,6 +19,8 @@ import { ViewClassTutorComponent } from './components/tutor/view-class-tutor/vie
 import { ErrorPageComponent } from './components/error-page/error-page.component';
 import { ConstructionPageComponent } from './components/construction-page/construction-page.component';
 import { LoginComponent } from './components/login/login.component';
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
```